pkgs/applications/window-managers/sway/contrib.nix

```diff
@@ -14,13 +14,21 @@
 , python3Packages
 }:
 
+let
+  version = "unstable-2023-06-30";
+  src = fetchFromGitHub {
+    owner = "OctopusET";
+    repo = "sway-contrib";
+    rev = "7e138bfc112872b79ac9fd766bc57c0f125b96d4";
+    hash = "sha256-u4sw1NeAhl4FJCG2YOeY45SHoN7tw6cSJwEL5iqr0uQ=";
+  };
+in
 {
 
 grimshot = stdenv.mkDerivation rec {
-  pname = "grimshot";
-  version = sway-unwrapped.version;
+  inherit version src;
 
-  src = sway-unwrapped.src;
+  pname = "grimshot";
```

Comment:
nit/comment: I would usually keep `pname` at the top for readability (and to keep this diff smaller) but it's kinda irrelevant.

Reply:
i feel good about the inherit line taking precedence over pname
or at least, having an inherit line in the middle of a mkDerivation would look worse to me